### PR TITLE
Server no app-root-path

### PR DESCRIPTION
**Pull request by gjedlicska**

### CODE REVIEW

File: packages/server/bootstrap.js [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
 require('module-alias')({ base: __dirname })
+const appRoot = __dirname
 
 // Initializing env vars
 const dotenv = require('dotenv')
 const { isTestEnv } = require('./modules/core/helpers/envHelper')
-const appRoot = require('app-root-path')
 
 // If running in test env, load .env.test first
 // (appRoot necessary, cause env files aren't loaded through require() calls)
```
Comment (fabis94) [inline packages/server/bootstrap.js:15]: Maybe we can get rid of appRoot here and just use `require.resolve` to resolve the path to the file, like you've done elsewhere 🤔

File: packages/server/scripts/duplicateUserMigration.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
-const appRoot = require('app-root-path')
-const knex = require(`${appRoot}/db/knex`)
-const roles = require(`${appRoot}/modules/core/roles.js`)
+const knex = require('@/db/knex')
+const roles = require('@/modules/core/roles.js')
```
Comment (fabis94) [inline packages/server/scripts/duplicateUserMigration.js:2]: These server scripts could be converted to yargs commands, but up to you

**Review by fabis94 — APPROVED** [review]
Added a few comments about potential improvements, but looks good overall 👍

File: packages/server/scripts/duplicateUserMigration.js (hunk 2 above)
Comment (gjedlicska) [reply packages/server/scripts/duplicateUserMigration.js:2]: yeah i was too lazy to do it :)

File: packages/server/bootstrap.js (hunk 1 above)
Comment (gjedlicska) [reply packages/server/bootstrap.js:15]: yeah i've pused an, we'll need the appRoot in `server/modules/index.js` line 34 anyhow

**Pull request merged**